src/sass/base/vd-tweaks.scss

```diff
@@ -38,20 +38,13 @@
 ///
 //////////////////////////////////////////////
 
+// line length--> re: Buttericks Typography
+// https://practicaltypography.com/line-length.html
 .au-body p {
 	max-width: 35rem;
```

Comment:
There is a variable in the design system for this. I would use that variable or override it.

https://github.com/govau/uikit/blob/master/packages/core/src/sass/_globals.scss#L737